frontend/src/pages/SignupPage/index.module.css

```diff
@@ -1,21 +1,30 @@
 .root {
   display: flex;
   flex-direction: column;
+  flex-grow: 1;
   align-items: center;
-  min-height: 100vh;
-  padding-bottom: 60px;
 }
 
 .main {
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   align-items: center;
-  padding: 0 var(--main-container-spacing);
 }
 
 .logo {
   display: flex;
   justify-content: flex-end;
-  padding: 50px;
+  margin-bottom: 45px;
+}
+
+.backLinkInvisible {
+  visibility: hidden;
+}
+
+.buttonWrapper {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 60px;
```

Comment:
What about adding `padding-bottom: 60px` to the `MainContainer`?

If that works, you need to make sure to remove the spacing from other individual components.

Reply:
I think you can remove `buttonWrapper` altogether.

Reply:
We tried to remove it but it didn't work. So @jreis13 put display and justify-content only and that works!